cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/casLogoutView.jsp

```diff
@@ -18,14 +18,21 @@
     under the License.
 
 --%>
-<jsp:directive.include file="includes/top.jsp" />
+
+<%-- CAS Page for Logged Out with no Service --%>
+
+<jsp:directive.include file="includes/top.jsp"/>
 
 <div id="msg" class="success">
-    <h2><spring:message code="screen.logout.header" /></h2>
-    <p><spring:message code="screen.logout.success" /></p>
-    <p><spring:message code="screen.logout.security" /></p>
+    <h2><spring:message code="screen.logout.header"/></h2>
+    <br/>
+    <p><spring:message code="screen.logout.success"/></p>
+    <p><spring:message code="screen.logout.security"/></p>
 </div>
 
-<c:set var="alternativeBottomLogin" value="true"/>
+<c:set var="linkSignIn" value="true"/>
```

Comment:
For generic logged out page (when there is no service), show "Sign In" and "Back to OSF".